ci/run.sh

```diff
@@ -93,7 +93,19 @@ time ELECTRON_ENABLE_LOGGING=1 yarn test
 log-group-end
 
 if [[ "${BUILDKITE_BRANCH:-}" == "master" || -n "${BUILDKITE_TAG:-}" ]]; then
-  log-group-start "Packaging and uploading app binaries"
-  time yarn dist
-  log-group-end
+  if [[ "${BUILDKITE_AGENT_META_DATA_PLATFORM:-}" == "macos" ]]; then
+    log-group-start "Packaging, notarizing and uploading app binaries"
+    (
+      export NOTARIZE=true
+      export APPLE_ID="rudolfs@monadic.xyz"
+      export APPLE_ID_PASSWORD="@keychain:AC_PASSWORD"
+      export CSC_NAME="Monadic GmbH (35C27H9VL2)"
+      time yarn dist
+    )
+    log-group-end
+  else
+    log-group-start "Packaging and uploading app binaries"
+      time yarn dist
```

Comment:
I the indentation intentional?

Reply:
Nope: d495437d70ee29131982dae3e5e12259c98a6415.